README.zh-Hans.md

```
@@ -0,0 +1,658 @@
+
+如果你的软件包使用 `package.json` 中的 `types` 或 `typings` 关键字指定了类型，那么 npm 注册表就会像这样显示该软件包有可用的绑定：
+
+![image](https://user-images.githubusercontent.com/30049719/228748963-56fabfd1-9101-42c2-9891-b586b775b01e.png)
+
+如果还是找不到类型，只需查找软件包中的任何".d.ts "文件，然后用 `/// <reference path="" />` 手动将其包含在内即可
+
+### 支持周期
+
+Definitely Typed 仅在发布时间小于 2 年的 TypeScript 版本上测试软件包。
+
+<img src="docs/support-window.svg#gh-light-mode-only" style="width:100%">
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)